### DISCUSSION TITLE
v5.23.0

### DISCUSSION THREAD

**Discussion by richardwilkes** [richardwilkes/gcs#846] [Announcements]
**New Features**
- A weapon's effective ST can now be adjusted via a weapon bonus (#842)
- Added support for editing the hit location note fields in the web version of the sheet

**Bug Fixes**
- Fixed a crashing bug when trying to import certain images into the web sheet's portrait field (#841)
- The light theme was using the wrong colors for headers and selected text in tables (#843)
- Fixed some issues in the web version's empty field handling

<hr /><em>This discussion was created from the release <a href='https://github.com/richardwilkes/gcs/releases/tag/v5.23.0'>v5.23.0</a>.</em>

**Comment by TheFabulousIronChef** [comment]
Those are some great new features. Thanks!
